heroku/cli
# refactor(redis-v5): Move 'redis:info' to CLI

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001axF9GYAU/view
https://gus.lightning.force.com/lightning/r/a07EE00001axCz4YAE/view

This converts `redis:info`  and `redis:index` to oclif and updates its associated test. The original implementation actually duplicated the code on both places (perhaps this was before aliases existed? At any rate, I elected to just do it as an alias as part of the convesion.

### Testing

1) Run `./bin/run redis:info -a APP` with this branch pulled down with an app that has redis on it
2) Verify the command shows info about redis
3) Run `./bin/run redis:info -a APP` with this branch pulled down with an app that does _not_ have redis on it
4) Verify the command shows nothing
5) Compare the results with prod cli command and make sure things look 👍
6) Run 1-5 again with `./bin/run redis` instead of `./bin/run redis:info`
